packages/shared/components/molecules/AssetIcon.svelte

```diff
@@ -67,9 +68,14 @@
             </p>
         {/if}
     </div>
-    {#if !(showVerifiedBadgeOnly && asset.verification !== VerificationStatus.Verified)}
-        <span class="absolute flex justify-center items-center h-4 w-4 -bottom-0.5 -right-0.5">
-            <VerificationBadge verificationStatus={asset.verification} {large} />
+    {#if !(showVerifiedBadgeOnly && asset?.verification !== VerificationStatus.Verified)}
+        <span
+            class="
+                absolute flex justify-center items-center h-4 w-4 
+                {small ? '-bottom-1 -right-1' : '-bottom-0.5 -right-0.5'}
+            "
+        >
+            <VerificationBadge verificationStatus={asset?.verification} {large} {small} />
```

Comment:
This doesn't affect anything, but `small` isn't a prop on `VerificationBadge.svelte`

Reply:
```suggestion
            <VerificationBadge verificationStatus={asset?.verification} {large} />
```